lib/smooth/include/smooth/core/network/Wifi.h

```diff
@@ -94,15 +81,14 @@ namespace smooth::core::network
             static void publish_status(bool connected, bool ip_changed);
 
             bool auto_connect_to_ap = false;
-            bool connected_to_ap = false;
-            std::string host_name = "Smooth-Wifi";
             std::string ssid{};
-
             std::string password{};
-            static struct esp_ip4_addr ip;
 
             esp_netif_t* interface{ nullptr };
             esp_event_handler_instance_t instance_wifi_event{};
             esp_event_handler_instance_t instance_ip_event{};
     };
+
+    Wifi&
+    get_wifi();
```

Comment:
```suggestion
    Wifi& get_wifi();
```

Reply:
yes, changed my clang-format to not do this.